examples/js/loaders/GLTF2Loader.js

```diff
@@ -2652,6 +2652,11 @@ THREE.GLTF2Loader = ( function () {
 
 							}
 
+							//do not clone children as they will be replaced anyway
+							var children = group.children
+							group.children=[];
+							var clonedgroup = group.clone();
+							group.children=children;
```

Comment:
Instead of creating a new group, I think you could replace these four lines with:

```js
_node.copy( group, false );
```

and then at the bottom of the loop, just add children to `_node` as before:

```js
_node.add( child );
```